utils/src/ooinstall/cli_installer.py

```diff
@@ -773,23 +794,23 @@ def cli(ctx, unattended, configuration, ansible_playbook_directory, ansible_conf
 
     # If no playbook dir on the CLI, check the config:
     if not ansible_playbook_directory:
-        ansible_playbook_directory = oo_cfg.settings.get('ansible_playbook_directory', '')
+        ansible_playbook_directory = oo_cfg.deployment.variables.get('ansible_playbook_directory', '')
     # If still no playbook dir, check for the default location:
     if not ansible_playbook_directory and os.path.exists(DEFAULT_PLAYBOOK_DIR):
         ansible_playbook_directory = DEFAULT_PLAYBOOK_DIR
     validate_ansible_dir(ansible_playbook_directory)
-    oo_cfg.settings['ansible_playbook_directory'] = ansible_playbook_directory
+    oo_cfg.deployment.variables['ansible_playbook_directory'] = ansible_playbook_directory
     oo_cfg.ansible_playbook_directory = ansible_playbook_directory
     ctx.obj['ansible_playbook_directory'] = ansible_playbook_directory
 
     if ctx.obj['ansible_config']:
-        oo_cfg.settings['ansible_config'] = ctx.obj['ansible_config']
-    elif 'ansible_config' not in oo_cfg.settings and \
+        oo_cfg.deployment.variables['ansible_config'] = ctx.obj['ansible_config']
+    elif 'ansible_config' not in oo_cfg.deployment.variables and \
```

Comment:
ansible_config and ansible_logpath should also be root level and not under deployments.